src/components/Home.js

```diff
@@ -6,8 +6,10 @@ import Form from './Form';
 const Home = () => (
   <div className="wrapper">
     <Navbar />
-    <Books />
-    <Form />
+    <div className="books-and-form">
+      <Books />
```

Comment:
- [ ] Styling of your app should be EXACTLY the same as the one provided in Zeplin. Color, Font size, Font weight, layout, and orientation should be pretty much the same as the one in the screenshot below:


Your Implementation | Zeplin Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/169254173-da6614df-f34c-4a9d-8fb1-beb9cf7b3d15.png) | ![](https://user-images.githubusercontent.com/63087491/169254200-7ab960f8-4298-42f7-8f82-5b2f2438b671.png)

- Please position your logo to align with the main container as you have done with the user icon.
- Your Progressbar and the `50% completed` is really too big.
- There's a vertical line between your Progressbar component and the chapter section.
- Please kindly take a look at the annotated screenshot above and try to make the necessary changes while comparing with the image from Zeplin design.